app/components/fields.js

```diff
@@ -2,25 +2,10 @@ import React, { Component } from "react";
 import { StyleSheet, View } from "react-native";
 
 import { Text } from ".";
+import { baseStyles } from "../styles";
+import FontAwesome, { Icons } from "react-native-fontawesome";
 
-const styles = StyleSheet.create({
-  field: {
-    borderBottomWidth: 1,
-    borderBottomColor: "#ccc",
-    backgroundColor: "white",
-    padding: 5,
-    flexDirection: "row",
-    justifyContent: "space-between",
-    alignItems: "center"
-  },
-  fieldLabel: {
-    fontSize: 10
-  },
-  fieldValue: {
-    color: "#aaa"
-  },
-  fieldArrow: {}
-});
+const styles = StyleSheet.create({});
```

Comment:
we might as well remove these empty styles objects and remove the `StyleSheet` imports from the top of the files where we're not using local styles.